ui/src/components/executions/Metrics.vue

```diff
@@ -1,3 +1,36 @@
+<script setup lang="ts">
+import { ref, nextTick } from 'vue';
+import ChartAreaspline from 'vue-material-design-icons/ChartAreaspline.vue';
+import Drawer from '../Drawer.vue';
+import MetricsTable from './MetricsTable.vue';
+
+// Props
+interface Props {
+    embed?: boolean;
+    taskRun: Record<string, any>;
+    execution: Record<string, any>;
+}
+
+const props = defineProps<Props>();
```

Comment:
In this codebase, we prefer to have as limited as possible number of single use interfaces. Specially confusiong when there name is shared with every component in the app.

```suggestion
const props = defineProps<{
    embed?: boolean;
    taskRun: Record<string, any>;
    execution: Record<string, any>;
}>();
```

Reply:
I have made the changes according to your suggestion.